l10n_br_cnpj_search/wizard/partner_cnpj_search_wizard.py

```diff
@@ -10,6 +10,7 @@
 
 class PartnerCnpjSearchWizard(models.TransientModel):
     _name = "partner.search.wizard"
+    _description = "Wizard searches partner data to update and edit."
```

Comment:
eu acho que o Inglês poderia ser mais idiomático. Que tal algo como "CNPJ based search wizard allowing to update partner data"?

Reply:
@rvalyi 
Alteração feita obrigado pela revisão.